package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gofynd/fdk-cli",
-  "version": "2.4.7",
```

Comment:
Include `https-localhost` package in package.json. [Reference](https://github.com/gofynd/fdk-cli/pull/144/files)